src/core/tests/logger.cpp

```diff
@@ -0,0 +1,142 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <iostream>
+#include <regex>
+#include <sstream>
+
+#include "openvino/core/log.hpp"
+#include "openvino/core/log_util.hpp"
+#include "openvino/util/log.hpp"
+
+namespace ov::util::test {
```

Comment:
Detail to check:
The test should be in `ov::tests`, the `ov::util::test` I think is used for test utils.